jobs/otel-collector-windows/templates/config.yml.erb

```diff
@@ -1,12 +1,37 @@
 <%=
 def config
   @config ||= begin
-    cfg = retrieve_property('config')
+    cfg = retrieve_config
     cfg = handle_old_properties(cfg) if cfg.empty?
     cfg
   end
 end
 
+def retrieve_config
+  cfg = p('config')
+
+  return cfg if cfg.respond_to?(:each)
+
+  unused_secrets = []
+  p('secrets').each do |secret|
+    %w[cert key ca secret].each do |type|
+      next if secret[type].nil? || secret[type].empty?
```

Comment:
I'd prefer to chain a .reject {|secret| secret[type].nil? || secret[type].empty? } instead of use `next`

Reply:
to be clear this is a stylistic preference, feel free to merge without changing it

Reply:
Going to ignore it for now, thanks!